db/seeds.rb

```diff
@@ -70,8 +67,10 @@
 
   puts 'Creating additional users ...'
 
+  number_of_users = (ENV['SEED_USERS'] || SEED_USERS).to_i
```

Comment:
I'd like to prefix the variables we use with "SHF_", so I'd like this one to be `SHF_SEED_USERS`

And this needs to be added to `.env.example`  perhaps with something like this in the

 `# These are optional environment variables:` section:`

```
# SHF_SEED_USERS=25     # The number of users to create during seeding.  (Takes precedence of the default number in seeds.db)

```